M66/M66ATParser/M66ATParser.cpp

```diff
@@ -178,6 +166,7 @@ bool M66ATParser::send(int id, const void *data, uint32_t amount)
     //May take a second try if device is busy
     for (unsigned i = 0; i < 2; i++) {
         if (tx("AT+QISEND=%d,%d", id, amount)
+            && rx(">")
```

Comment:
Looks like the docs were unclear whether we need the > prompt for fixed length data.